src/devices/machine/mm5740.cpp

```diff
@@ -51,13 +51,12 @@ mm5740_device::mm5740_device(const machine_config &mconfig, const char *tag, dev
 	m_write_data_ready(*this),
 	m_rom(*this, "internal")
 {
-	std::fill(std::begin(m_x_mask), std::end(m_x_mask), 0);
 }
 
-uint32_t mm5740_device::calc_effective_clock_key_debounce(uint32_t capacitance)
+u32 mm5740_device::calc_effective_clock_key_debounce(u32 capacitance)
 {
 	// calculate key debounce based on capacitance in pF
-	uint32_t key_debounce_msec = capacitance / 125;
+	u32 key_debounce_msec = capacitance / 125;
```

Comment:
The rest of this file uses the POSIX-style `uint32_t`, `uint16_t`, etc.  Please don’t change it to mix POSIX-style and abbreviated style.

Reply:
There was only one remaining posix style in this file, the constructor, I'll update that to be consistent with the rest of the file.